lib/wiki_extensions_child_pages_count_macro.rb

```diff
@@ -0,0 +1,14 @@
+require 'redmine'
+
+module WikiExtensionsChildPagesCountMacro
+  Redmine::WikiFormatting::Macros.register do
+    desc "Displays the number of child pages.\n\n" +
+           "{{child_page_count}}\n"
+    macro :child_pages_count do |obj, args|
+      return t(:error_child_pages_count_macro) unless obj.is_a?(WikiContent)
+
+      child_pages = obj.page.descendants
+      child_pages.size
```

Comment:
We should return a `String`.

Reply:
fix: [b2b6ede](https://github.com/otegami/redmine_wiki_extensions/pull/1/commits/b2b6ede7fa75a349bed88f953fa25ac2be7e426e): This macro used in Wiki should return `String` because it will be used in text content.